### ISSUE TITLE
env file not being created / GITHUB_WORKSPACE not set

### ISSUE THREAD

**Issue by speclechris** [SpicyPizza/create-envfile#45]
```
name: Create envfile

on: [ push ]

jobs:
  create-envfile:
    runs-on: ubuntu-latest
    steps:
    - name: Make envfile
      uses: SpicyPizza/create-envfile@v1.3
      with:
        envkey_SOMETHING: something
```

I expected this to create the env file within my project directory e.g. `/home/runner/work/application-name/application-name/.env`

But I get the following:
`Creating file: /github/workspace/.env`

Only thing I can guess is GITHUB_WORKSPACE isn't set and it's defaulting to this path? 
`path = str(os.getenv("GITHUB_WORKSPACE", "/github/workspace"))`

**Comment by julesmqzsma** [comment]
I have the same issue. how can we configure the path? Directory property seems to not be able to overwrite the full path

**Comment by JuniorMiksza8** [comment]
I have the same issue, any solution so far ?

**Comment by rorlich** [comment]
me 2

**Comment by ianrussel** [comment]
me too

**Comment by lclarkg18** [comment]
Same issue here

**Comment by hayder-uruit** [comment]
Same issue here!

**Comment by hayder-uruit** [comment]
I recommend using the version SpicyPizza/create-envfile@v1.2

**Comment by cronnoss** [comment]
Same issue here!

**Comment by AngelOnFira** [comment]
This should be resolved in v2.0.1 but do mention if the issue still persists!
